action-bump-version/action.yaml

```diff
@@ -0,0 +1,75 @@
+name: Bump patch version
```

Comment:
generally i wonder about this action:
* about the name
* about the reusability

At the moment, it seems very specific to one approach, i.e. artesca's VERSION property file, while the name is very specific.

Having such an action would be nice indeed, but likely not restricted to artesca: e.g. we would want to do the same with Zenko, which has a different approach (and path) for the VERSION file ; and also to bump the version of dependencies, e.g. updating artesca's PROJECT_VERSION ; or even updating zenko's deps.yaml (though that is even more different...)

Starting small is fine, but we should try to think ahead to avoid issues coming later on...